db/migrate/20221208013832_add_shopper_to_marketplace_carts.rb

```diff
@@ -0,0 +1,12 @@
+class AddShopperToMarketplaceCarts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :marketplace_shoppers, id: :uuid  do |t|
+      t.references(:person, type: :uuid, foreign_key: {to_table: :people}, index: {unique: true})
```

Comment:
Does this `unique: true` mean that we intend there to only ever be at most one Shopper linked to a Person, among all Marketplaces? Or is this also scoped by Marketplace, and I'm just not seeing it?